public/gameList.php

```diff
@@ -54,6 +54,7 @@ function ListGames(
         echo "<th class='text-right'><a href='/gameList.php?s=$sort2$queryParams'>Achievements</a></th>";
         echo "<th class='text-right'><a href='/gameList.php?s=$sort3$queryParams'>Points</a></th>";
         echo "<th class='text-right'><a href='/gameList.php?s=$sort7$queryParams'>Retro Ratio</a></th>";
+        echo "<th style='white-space: nowrap'><a href='/gameList.php?s=$sort6$queryParams'>Last Updated</a></th>";
```

Comment:
This should be right aligned.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/bca2f13b-ff47-4e27-a661-68936028dc92)

Reply:
Thanks, the game list I was testing against had long enough game names to push the table to the point that I missed this. Resolved in https://github.com/RetroAchievements/RAWeb/pull/1841/commits/6d99280826b7804a024918739024ca28899f7e71.